seatunnel-connectors-v2/connector-file/connector-file-base-hadoop/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/hdfs/source/BaseHdfsFileSource.java

```diff
@@ -110,13 +112,26 @@ public void prepare(Config pluginConfig) throws PrepareFailException {
                             "SeaTunnel does not supported this file format");
             }
         } else {
-            try {
-                rowType = readStrategy.getSeaTunnelRowTypeInfo(hadoopConf, filePaths.get(0));
-            } catch (FileConnectorException e) {
+            FileConnectorException fileConnectorException = null;
```

Comment:
When will there be an empty ORC file in the directory? Even if the ORC file is empty, I believe the meta area of ORC still has value. Can you provide an E2E case to verify whether this modification takes effect?

Reply:
Sometimes during data migration, some such files will be generated due to the network or hadoop system itself, but hadoop itself will not take the initiative to clean up。